easy_rec/python/tools/faiss_index_pai.py

```diff
@@ -0,0 +1,112 @@
+# -*- encoding:utf-8 -*-
+# Copyright (c) Alibaba, Inc. and its affiliates.
+from __future__ import print_function
```

Comment:
a document is required to introduce:
1. how to build index
2. how to deploy models